addons/event_sale/views/event_views.xml

```diff
@@ -5,8 +5,9 @@
         <field name="name">event.form.inherit</field>
         <field name="model">event.event</field>
         <field name="inherit_id" ref="event.view_event_form"/>
+        <field name="priority" eval="8"/>
```

Comment:
For the priorities, I would use bigger number just in case one day we want to "insert" a view between 2 other priorities
(e.g. 10-20-30-40 so if one day I want to add a view between 10 and 20 I'll have only 1 line diff)